components/automate-deployment/pkg/target/local_target.go

```diff
@@ -1520,3 +1520,22 @@ func (t *LocalTarget) IPs() []net.IP {
 func (t *LocalTarget) HabCache() depot.HabCache {
 	return depot.FromLocalCache()
 }
+
+func (t *LocalTarget) InstallAutomateBackendDeployment(ctx context.Context, m manifest.ReleaseManifest) error {
+	//var pkg habpkg.HabPkg = habpkg.New("chef", "automate-backend-deployment")
+	pkg := manifest.InstallableFromManifest(m, "automate-backend-deployment")
+	if pkg == nil {
+		logrus.Info("(HA) unable to find automate-backend-deployment package in manifest")
+		return errors.New("automate-backend-deployment (HA) was not found in the manifest")
+	}
+	output, err := t.InstallPackage(ctx, pkg, "dev")
```

Comment:
we can change dev -> c.GetV1().GetSvc().GetChannel().GetValue()
once automate will release next week